server/app/views/components/TextFormatter.java

```diff
@@ -50,6 +51,8 @@ public enum UrlOpenAction {
   private static final String ACCORDION_CONTENT = ">";
   private static final String ACCORDION_HEADER = "### ";
   private static final String BULLETED_ITEM = "* ";
+  private static final char HEADER = '#';
```

Comment:
Same question as I left in the typescript. What is this indicating?

Reply:
I explained above, but lmk if you have more questions about this one in particular.